ra2ce/common/validation/validation_report.py

```diff
@@ -25,6 +25,9 @@
 
 
 class ValidationReport:
+    _errors: list
+    _warns: list
```

Comment:
you might as well add the type of `list` items (`list[str]` if I recall correctly)

Reply:
Done